panel/config/addplugindialog.cpp

```diff
@@ -49,7 +49,11 @@ AddPluginDialog::AddPluginDialog(QWidget *parent):
     ui->setupUi(this);
 
     QStringList desktopFilesDirs;
+#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))
```

Comment:
We neither support previous releases of LXQt nor backport new features to them. We need to make our life easier (there are too few people maintaining lxqt). 

Do we want to mess our code with these ugly #ifs? Why not just require v5.15?